drivers/dstate.c

```diff
@@ -1167,7 +1167,7 @@ int dstate_detect_phasecount(
 
 		if ( (v1 && v2 && !v3) ||
 		     (v1n && v2n && !v3n) ||
-		     (c1 && c2 && !c2) ||
+		     (c1 && c2 && !c3) ||  // •••Sean••• not sure...
```

Comment:
yes, those are bullets. the comment is not intended to ever get committed though.

Reply:
@jimklimov aren't you the author of that code?! ;-p
confirming that @seanm is right, for dual/split phase UPS, we have x1 && x2 (x being voltage, current, for phase 1 and 2), and not x3 for phase 3, otherwise, it would be a 3 phase unit

Reply:
OK, with that confirmation, I'll remove the temporary comment.